### DISCUSSION TITLE
Web User interface: The differentiation in the display of the number of posts on the profile page

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#18008] [General]
Personally, I find the number of posts on the public profile page a bit confusing at the moment, because as far as I can see, it shows public, semi-public, non-public posts and direct messages (please correct me if this is not the case). So the total number is probably made up of these. 

I would find it useful to differentiate at least between public and semi-public posts and possibly also between follower-only posts (non-public posts). Direct messages, if they are included, would only be displayed for themselves and completely deleted from the public count, if they are counted at all. How do people from the community see it? Similar? Opposite?

**Comment by aiaustin** [comment]
I also found that Direct Messages (limited to users named in the Toot) were not differentiated in the timeline from other public Toots. A "Scope of Toot" marker in the Toot, and perhaps even a different colour background for DM/limited scope Toots would be very helpful I think.  It also would make clear the scope you used in case a mistake was made in selecting that.

The @Direct Messages" area of the web/app interface could also usefully have a "New Toot" initiation button which opens the compose Toot pane with the scope already selected as DM/Direct. But maybe that should be a separate issue.

**Reply by shoqvalue** [reply]
LOL I was just about to ask Gargron of this, and thought I would check here first.  Here is the post I was composing:

> Doesn't anyone else get completely panicked when they see their DMs in notifications and the home timeline, but with no visual cues to tell you that they are  private (a border, background, or something!)?  I cannot be the only one creeped out by that.  I assume it's been discussed. I'll go check github now. This is unnerving enough for power users. Image the heart failure rate among newbies just getting into this the first time they fear that maybe their private discussion just went viral :)  I don't mean to make light of it. It's just scary, and not a very nice user experience. I'm surprised it's gone this long without discussion.
